### PR TITLE
go tools cover moved in Go 1.4*

### PR DESCRIPTION
**Pull request by fabianofranz**

### CODE REVIEW

**Comment by fabianofranz** [comment]
@smarterclayton

File: images/release/Dockerfile [hunk 1]
```diff
@@ -11,7 +11,8 @@ RUN yum install -y hg golang golang-pkg-darwin golang-pkg-windows && yum clean a
 ENV GOPATH /go
 
 # Get the code coverage tool and godep
-RUN go get code.google.com/p/go.tools/cmd/cover github.com/tools/godep github.com/golang/lint/golint
+RUN if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
```
Comment (ncdc) [inline images/release/Dockerfile]: Why are you checking the old location that's no longer valid?

Comment (ncdc) [inline images/release/Dockerfile]: Is this really a go 1.4 change?

Comment (fabianofranz) [inline images/release/Dockerfile]: The old location is still valid and recommended for Go 1.2/1.3. Starting in 1.4 the new location must be used. Related upstream issue: https://github.com/GoogleCloudPlatform/kubernetes/issues/2728. 

So I tried to use the same strategy used in `travis.yml`, but I think we can safely remove the condition and just use the new location if we are guaranteed to always get 1.4 from `yum install golang`.

**Comment by liggitt** [comment]
We're building successfully with 1.4 now. Is this necessary?

**Pull request closed without merging**